Source/WebCore/html/canvas/WebGL2RenderingContext.cpp

```diff
@@ -591,11 +605,12 @@ void WebGL2RenderingContext::copyBufferSubData(GCGLenum readTarget, GCGLenum wri
         return;
 
     RefPtr<WebGLBuffer> readBuffer = validateBufferDataParameters("copyBufferSubData", readTarget, GraphicsContextGL::STATIC_DRAW);
+    if (!readBuffer)
+        return;
```

Comment:
Is the "Invalid readTarget" synthetic error not required here?

Reply:
`validateBufferDataParameters` or its callees generate a synthetic error before returning `nullptr`.